### ISSUE TITLE
Scram authentication error when running seed-jaffle.sh

### ISSUE THREAD

**Issue by rephus** [lightdash/lightdash#1558]
### Describe the bug

I was following  the guide on https://github.com/lightdash/lightdash/blob/main/.github/CONTRIBUTING.md#setup-development-environment, when I tried to run `./docker/scripts/seed-jaffle.sh` I got the following error

![image](https://user-images.githubusercontent.com/1983672/159248316-70bbef80-c5bd-4813-845c-8f1f0653f03b.png)

However, the version of libpq is on 13.5 

![image](https://user-images.githubusercontent.com/1983672/159248416-97dba642-39f7-4a17-ae18-194c68d6266f.png)

I found this post about the same issue 

https://forums.docker.com/t/psycopg2-operationalerror-scram-authentication-requires-libpq-version-10-or-above-python-3-7-slim/119830

There it suggests to downgrade the version of libpq in pip , that fixed the issue for me too. 

```pip install psycopg2-binary==2.8.6```

![image](https://user-images.githubusercontent.com/1983672/159248613-5026374c-a9ce-4029-bba9-73275706b3c1.png)

However, it says it might affect performace, so this fix requires a bit more investigation. 

### To Reproduce

1. Follow the CONTRIBUTING guide and build a new dev environment
2. Run docker compose
3. Follow the guide until you reach `./docker/scripts/seed-jaffle.sh`

### Expected behavior

`./docker/scripts/seed-jaffle.sh` should run without errors

**Comment by 12ian34** [comment]
Have also just experienced this issue. downgrading libpq in pip also worked for me

**Comment by JohnRomanski** [comment]
I encountered this issue as well, **when building on an M1 MacBook Pro**
```
14:11:12  Encountered an error:
Database Error
  SCRAM authentication requires libpq version 10 or above
```
My google results lead me to a different solution. https://stackoverflow.com/a/70238851

As a result, I added the suggested configuration into the `docker-compose.dev.yml` file.

```
  lightdash-dev:
+   platform: linux/amd64
...
  db-dev:
+   platform: linux/amd64
```

Then I rebuilt the images completely for the new architecture, and the error went away. Two line PR incoming.
